netdcli/netdcli.go

```diff
@@ -498,26 +508,53 @@ func executeOpts(opts *cliOpts) error {
 	case cliOperGet:
 		err = coreState.Read(opts.idStr)
 		if err != nil {
-			log.Fatalf("Failed to read %s. Error: %s", opts.construct.Get(), err)
-		} else {
-			content, err := json.MarshalIndent(coreState, "", "  ")
-			if err != nil {
-				log.Fatalf("Failed to marshal corestate %+v", coreState)
-			}
-			fmt.Println(string(content))
+			log.Errorf("Failed to read %s. Error: %s", opts.construct.Get(), err)
+			return err
 		}
+		content, err := json.MarshalIndent(coreState, "", "  ")
```

Comment:
If we're going to use this MarshalIndent (with no leading spaces and a 2-space indent) regularly, we should make it a function out of it.